theme_clean/views/snippets/s_company_team.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team" inherit_id="website.s_company_team">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt0 pb16" remove="pt48 pb48" separator=" "/>
```

Comment:
Is the padding top is what we want?
![image](https://github.com/user-attachments/assets/9a8f3629-2e41-4f8f-8bd2-2f7d47090b7e)

Reply:
ok snippet by default has now 0 custo (thus is pt48 pb48 padding), but template for configurator has now the changes I want for the page that will come out from the configurator.